src/components/Link/Link.tsx

```diff
@@ -86,9 +88,9 @@ export const Link = <T extends ElementType = "a">({
     {hasIcon && (
       <IconWrapper $size={size}>
         <Icon
-          name="popout"
+          name={icon ?? "popout"}
```

Comment:
storybook needs to be updated as well

Reply:
https://click-ui-git-custom-icon-for-links-clickhouse.vercel.app/?path=%2Fdocs%2Ftypography-link--docs
Its there in storybook